pinakes/main/common/views.py

```diff
@@ -26,6 +26,13 @@
     ),
     list=extend_schema(
         description="List all groups",
+        parameters=[
+            OpenApiParameter(
+                "role",
+                type={"type": "array", "items": {"type": "string"}},
+                description="Any RBAC roles the groups belong to",
```

Comment:
@bzwei Since this is the confusing bit should be put an example in the description e.g. role=A&role=B&role=C